genderbias/effort/__init__.py

```diff
@@ -53,58 +55,79 @@ def get_report(self, doc):
 
         """
         report = Report("Effort vs Accomplishment")
-        effort_flags = []
-        accomplishment_flags = []
-
-        token_indices = doc.words_with_indices()
-
-        for word, start, stop in token_indices:
-            if word.lower() in EFFORT_WORDS:
-                effort_flags.append(
-                    Flag(
-                        start,
-                        stop,
-                        Issue(
-                            "Effort vs Accomplishment",
-                            f"The word '{word}' tends to speak about "
-                            + "effort more than accomplishment.",
-                            "Try replacing with phrasing that emphasizes accomplishment.",
-                            bias=Issue.negative_result,
-                        ),
-                    )
-                )
-            if word.lower() in ACCOMPLISHMENT_WORDS:
-                accomplishment_flags.append(
-                    Flag(
-                        start,
-                        stop,
-                        Issue(
-                            "Effort vs Accomplishment",
-                            f"The word '{word}' tends to speak about "
-                            + "accomplishment more than effort.",
-                            bias=Issue.positive_result,
-                        ),
+
+        nlp = spacy.load("en_core_web_sm")
+        doc = nlp(doc.text())
+
+        # Ignore adjectives about the author.
+        # TODO: This is dependent upon the type of writing! If this is a cover-
+        # letter, e.g., then these are not good to exclude!
```

Comment:
This seems like more of a NOTE than a TODO. The project overall isn't doing much with genré yet; it's not clear whether the scope will ever include cover letters. Unless @molliem says it will. :-)

(Sorry, @j6k4m8 , you've been pushing one my buttons - I really dislike TODO comments. I'll let this go and approve your PR unless I find something more significant that needs changing.)